.changeset/slimy-cougars-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/widgets': patch
+---
+
+Add react and react hooks recommended eslint config
```

Comment:
This kind of PR doesn't need a changeset because it doesn't affect the consumers of this lib in any way